packages/forms-web-app/__tests__/unit/controllers/projects/examination-timetable/utils/events/event/get-event.component.test.js

```diff
@@ -0,0 +1,39 @@
+const {
+	getEvent
+} = require('../../../../../../../../src/controllers/projects/examination-timetable/utils/events/event/get-event');
+
+describe('controllers/projects/examination-timetable/utils/events/event/get-event', () => {
+	describe('#getEvent', () => {
+		describe('When getting the data for an event', () => {
+			let result;
+			const mockEvent = {
+				dateOfEvent: '2023-01-02',
```

Comment:
these dates should be const and tell us what they are. 

i/e - today, tomorrow, yesterday, leap year, 1 month ago etc. 

Will help when we come back to them